### PR TITLE
This allows user to set the surface positions for the aeroterminate

### PR DESCRIPTION
**Pull request by QuPsi**
Values can vary from -100% to 100% deflection.

### CODE REVIEW

**Comment by peterbarker** [comment]
What's the use case here?

**Comment by peterbarker** [comment]
I've rebased this branch as it was wedged in CI.

File: libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp [hunk 1]
```diff
@@ -158,6 +158,27 @@ const AP_Param::GroupInfo AP_AdvancedFailsafe::var_info[] = {
     // @Units: km
     AP_GROUPINFO("MAX_RANGE",   20, AP_AdvancedFailsafe, _max_range_km,    0),
     
+    // @Param: TERM_POS_AIL
+    // @DisplayName: Aileron Aeroterminate Position
+    // @Description: This is the position the aileron will go to during an aerotermination
+    // @User: Advanced
+    // @Units: Percentage (-100 to 100, default 100)
+    AP_GROUPINFO("TERM_POS_AIL",   23, AP_AdvancedFailsafe, _term_pos_ail,    100),
```
Comment (magicrub) [inline libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp:166]: index should be 21.
Units should be ```// @Units: %```
Add ```// @Range: -100 100```
Description should mention that 0 means neutral deflection and could also mention the default
Display name would be better to say ``Aeroterminate Position for Aileron'' so Aileron is at the end so the param names and descriptions only have that one attribute different on the end.

File: libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.h [hunk 2]
```diff
@@ -131,6 +131,9 @@ class AP_AdvancedFailsafe
     AP_Int8  _rc_term_manual_only;
     AP_Int8  _enable_dual_loss;
     AP_Int16  _max_range_km;
+    AP_Int8  _term_pos_ail;
```
Comment (magicrub) [inline libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.h:134]: the other terminate variables start with ``_terminate_`` so please follow that pattern.

File: libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp (hunk 1 above)
Comment (magicrub) [reply libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp:166]: Also, the word "Aeroterminate" is new. Let's keep the vocabulary consistent and name it ```Terminate Aileron Position```

**Review by magicrub — CHANGES_REQUESTED** [review]

**Comment by tridge** [comment]
we need some good justification for why these parameters are needed

**Comment by auturgy** [comment]
The params should be hidden under AFS_ENABLE, so not visible to most users.

**Comment by QuPsi** [comment]
In some aircraft, because of channel reversal the aircraft will pitch up instead of down during aeroterminate. This allows the user to set the positions for the aeroterminate. I am reviewing the other comments and will do my best incorporate them.

**Comment by QuPsi** [comment]
> The params should be hidden under AFS_ENABLE, so not visible to most users.

How do I accomplish this?

**Comment by magicrub** [comment]
> > The params should be hidden under AFS_ENABLE, so not visible to most users.
> 
> How do I accomplish this?

I think @auturgy was just telling @tridge that this param will be hidden when ADFS_ENABLE=0 via AP_PARAM_FLAG_ENABLE so this new param won't even show up for most users

**Comment by QuPsi** [comment]
Continued below:

https://github.com/ArduPilot/ardupilot/pull/15458

**Pull request closed without merging**